Unable to import PropertyFieldGrid control

#### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question

#### Version
[3.19.0]

_If you are not using the latest release, please update and see if the issue is resolved before submitting an issue._

Installed the latest but still didn't resolve the issue

#### Expected / Desired Behavior / Question
Able to use PropertyFieldGrid control normally after install the latest library version

#### Observed Behavior
Get an error stating "Cannot find module '@pnp/spfx-property-controls/lib/PropertyFieldGrid' or its corresponding type declaration". It must be missing in some index export somewhere

#### Steps to Reproduce
1. On the top of the file, attempt to import the PropertyFieldGrid control
2. import { PropertyFieldGrid } from "@pnp/spfx-property-controls/lib/PropertyFieldGrid";

--- a/src/PropertyFieldGrid.ts
+++ b/src/PropertyFieldGrid.ts
@@ -1,0 +1,1 @@
+export * from './propertyFields/propertyFieldGrid/index';

--- a/src/index.ts
+++ b/src/index.ts
@@ -30,3 +30,4 @@ export * from './PropertyFieldColumnPicker';
 export * from './PropertyFieldMonacoEditor';
 export * from './PropertyFieldMessage';
 export * from './PropertyFieldButton';
+export * from './PropertyFieldGrid';

